lms/lib/courseware_search/lms_result_processor.py

```diff
@@ -73,7 +73,10 @@ def get_display_name(category, item_id):
 
         def get_position_name(section, position):
             """ helper to fetch name corresponding to the position therein """
-            pos = int(position)
+            if position is None:
+                return None
+
+            pos = int(position.split('_', 1)[0])
```

Comment:
This seems fragile to me -- we're assuming 1) position is a string, 2) it contains an underscore, and 3) the item split-off is an integer.  I would at least toss a try/except around the int() cast unless it's better to raise the exception for this use case.  Thoughts?